### PR TITLE
#234: Merge 234 to training

### PR DESCRIPTION
**Pull request by ga-hegsvold**
Architecture & Concepts: Beginner - Datasets

### CODE REVIEW

File: training/010_architecture_and_concepts/020_Beginner.rst [hunk 1]
```diff
@@ -524,15 +524,38 @@ Where to make new ref 1.1.6
 Datasets
 ~~~~~~~~
 
-Something more general about pipes maybe in context of systems and pipes
+Datasets are where data is stored inside Sesam, regardless of whether the
+data comes from external systems or from internal pipes.
 
-Very low level but enough to see entities?
+Data in a dataset is represented as a JSON list where each list item is a
+data record, called *entity*, consisting og key-value pairs.
```
Comment (JonasAls89) [inline training/010_architecture_and_concepts/020_Beginner.rst:531]: typo "og" -< of.

File: training/010_architecture_and_concepts/020_Beginner.rst [hunk 2]
```diff
@@ -524,15 +524,38 @@ Where to make new ref 1.1.6
...
 
-and refer to entities subchapter ref. 1.1.12
+A dataset with two people entities could look something like this:
```
Comment (JonasAls89) [inline training/010_architecture_and_concepts/020_Beginner.rst:533]: I would rephrase this, "people entities" comes across as a specific type of entity, i.e.: A dataset with two entities concerning persons could look like this:

**Review by JonasAls89 — CHANGES_REQUESTED** [review]

File: training/010_architecture_and_concepts/020_Beginner.rst (hunk 1 above)
Comment (ga-hegsvold) [reply training/010_architecture_and_concepts/020_Beginner.rst]: fixed

File: training/010_architecture_and_concepts/020_Beginner.rst (hunk 2 above)
Comment (ga-hegsvold) [reply training/010_architecture_and_concepts/020_Beginner.rst]: fixed

**Review by JonasAls89 — APPROVED** [review]

**Review by gabriellsesam — APPROVED** [review]
